dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dashboard/DashboardUserInfoService.java

```diff
@@ -27,8 +27,13 @@ public class DashboardUserInfoService {
     public List<DashboardUserInfo> createDashboardUsers(UserListResult userListResult) {
         List<DashboardUserInfo> userInfoList = new LinkedList<>();
         for (AuthenticatedUser user : userListResult.getUserList()) {
-            userInfoList.add(new DashboardUserInfo(user, getAuthProviderFriendlyName(user),
-                    hasSelectedConfirmedEmail(user), getUserNotificationLanguageDisplayName(user, session.getLocale())));
+            if (!user.isErased()) {
```

Comment:
Not sure this is the best place to filter. What about counters, are they still correct? I would have guessed that `UserServiceBean` or queries in `AuthenticatedUser` would be a better place to filter out erased accounts in order have a more consistent impact.

Reply:
fixed